staff/luis-garrido/gamma-template/about.html

```diff
@@ -0,0 +1,153 @@
+                    <img class="downloadApps__images--imageWidth" src="img/icon_app_store.png" />
+                    <br />
+                    <img class="downloadApps__images--imageWidth" src="img/icon_google_play.png" />
+                </div>
+            </div>
+            <div class="downloadApps__right">
+                <img class="downloadApps__phone" src="img/site_phone.png" />
+            </div>
+        </section>
+
+        <section class="news">
+            <article class="article">
+                <img class="article__image" src="img/demo1.jpg" />
+                <h2 class="article__title">scotcher</h2>
+                <p class="article__text">divulsor adaptative distressingly octuply Cyathophyllidae accreditment worldway
+                    prunably photogram minification dismayingly stigmatose wheelroad orchioplasty meaningness incurvature
+                    atweel oxidator locellus seattleble malarial porty unfleshed afara    
+                </p>
+                <a class="article__link" href="#">Read more</a>
+            </article>
+            <article class="article">
+                <img class="article__image" src="img/demo2.jpg" />
+                <h2 class="article__title">cabalistically</h2>
+                <p class="article__text">th scripula alcyonic secessionist underminable sexivalent bronchoalveolar dogy
+                    registrational mishmee cinderlike Encephalartos flow indetermine mosquish counterscarp revieweress
+                    inflectionless pelagic diamminobromide placement unfroward Spica uncontributing</p>
+                <a class="article__link" href="#">Read more</a>
+            </article>
+            <article class="article">
+                <img class="article__image" src="img/demo3.jpg" />
+                <h2 class="article__title">jargonium</h2>
+                <p class="article__text">inversedly ergosterol desightment bombastically tandemist panegyrical unstormy
+                    solicitee cassine arbitrarily starring unherolike caulicole virtueproof beveler stereomicrometer
+                    orvietan sliced abstinency spurtive coextensiveness prescient senicide unradicalize</p>
+                <a class="article__link" href="#">Read more</a>
+            </article>
+        </section>
+        
+        <section class="features">
+            <div class="features__left">
+                <h2 class="features__title">Gamma Features</h2>
+                <ul class="features__list">
+                    <li>charlatanistic</li>
+                    <li>unembattled</li>
+                    <li>bleekbok</li>
+                    <li>overtipple</li>
+                    <li>empathically</li>
+                </ul>
+            </div>
+            <div class="features__right">
+                <img class="features__image" src="img/logo.png" alt="camera icon" />
+            </div>
+        </section>
+    </main> -->
+    
+    <footer class="footer">
+        <ul>
```

Comment:
Maybe you could use a nav in the footer, just for semantic purpose.

Reply:
Done!